loops_solution.js

```diff
@@ -0,0 +1,116 @@
+//Question 1
+/*
+let num = 40;
+while (num > 1) {
+  console.log(num);
+  num --;
+}
+
+for (let num = 40; num > 1; num --) {
+  console.log(num);
+}
+
+//Question 2
+
+let num = 0;
+while (num < 50) {
+  if(num % 2) {
+    console.log(num + " is odd.")
+  } else {
+    console.log(num + " is even.")
+  }
+  num ++;
+}
+
+for (let num = 0; num < 50; num ++) {
+  if(num % 2) {
+    console.log(num + " is odd.")
+  } else {
+    console.log(num + " is even.")
+  }
+}
+
+//Question 3
+
+let num = 0;
+while (num < 50) {
+  console.log(num * 9);
+  num ++;
+}
+
+for (let num = 0; num < 50; num ++) {
+  console.log(num * 9);
```

Comment:
Hey! The only thing here is that we asked you to write out the math equation in your console.log - for example: `console.log(num + " * 9 = " + num * 9)`